server/src/middlewares/series.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash');
+
+const data = require('../data');
+
+/**
+ * @tested
+ * @param {object | null} root
+ * @param {object} args
+ * @param {object} context
+ * @param {object} info
+ * @return {Promise<void>}
+ */
+async function serie(root, args, context, info) {
```

Comment:
what is "serie"? The singular of "series"? If so, it could be "seriesSingle" or similar. Series is kind of a strange word, in that its singular and plural are the same.

Reply:
Thanks, will fix.

Reply:
Fixed.